docs/content/code-outputs.md

```diff
@@ -173,28 +175,27 @@ sphinx:
 ```
 
 (content:code-outputs:glue)=
-
 ## Insert code outputs into page content
 
-You often wish to run analyses in one notebook and insert them into your
-documents text elsewhere. For example, if you'd like to include a figure,
-or if you want to cite a statistic that you have run.
+You often wish to run analyses in one notebook and insert them in your
+documents elsewhere. For example, if you'd like to include a figure,
+or if you want to cite an analysis that you have run.
 
 ```{margin}
 Currently, `glue` only works with Python.
 ```
 
-The **`glue`** tool from [MyST-NB](https://myst-nb.readthedocs.io/)
+The `glue` tool from [MyST-NB](https://myst-nb.readthedocs.io/)
 allows you to add a key to variables in a notebook,
 then display those variables in your book by referencing the key. It
 follows a two-step process:
 
-* **Glue a variable into your notebook**. Do this by using
+* **Glue a variable to a name**. Do this by using
```

Comment:
This is an interesting distinction - it does semantically change what a user is "doing" when they use glue, but I kinda think that glueing something to a name is a more accurate way of thinking about it than glueing to the notebook. Just adding this comment here in case others have thoughts, but I think I'm fine w/ the change

Reply:
Thanks for noticing. It does mean something different but imho it makes more sense, hence my proposed change.